### PR TITLE
Dockerfile: Compile protoc-gen-swagger

### PR DESCRIPTION
**Pull request by htdvisser**
Include the protoc-gen-swagger binary

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -102,7 +103,7 @@ RUN mkdir -p /grpc-swift && \
         patchelf --set-interpreter /protoc-gen-swift/ld-linux-x86-64.so.2 /protoc-gen-swift/${p}; \
     done
 
-    
+
```
Comment (rvolosatovs) [inline Dockerfile:106]: Unrelated change. Please undo this.

**Review by rvolosatovs — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (htdvisser) [reply Dockerfile:106]: No.

**Comment by rvolosatovs** [comment]
The change is integrated in `3.0.11`
